pkgs/applications/networking/instant-messengers/franz/default.nix

```diff
@@ -3,13 +3,13 @@
 , gnome2, dbus, nss, nspr, alsaLib, cups, expat, udev, libnotify, xdg_utils }:
 
 let
-  version = "5.4.1";
+  version = "5.5.0-beta.1";
```

Comment:
We should wait for the stable release unless there is a good reason to switch to a pre-release.

Reply:
Meanwhile, 5.5.0 is out on 30 Apr.